src/services/authentication.service.js

```diff
@@ -64,11 +70,11 @@ export default class UserService {
    */
   static findUserByEmailOrUsername = async (value) => {
     let currUser = await user.findOne({
-      where: { username: value },
+      where: { username: value }, include: requestInclusionToUserRetrieval
     });
     if (!currUser) {
       currUser = await user.findOne({
-      where: { email: value },
+        where: { email: value }, include: requestInclusionToUserRetrieval
```

Comment:
@de-scholar , there is no need to retrieve a user's requests here. There is a different endpoint for that

Reply:
Today I will do a different endpoint for retrieving requests, thank you

Reply:
I implemented this suggestion